### DISCUSSION TITLE
Transfer ownership form

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#838] [Suggestions]
Add a form that folks can use to transfer ownership of their dashboard to another user

**Comment by Loops117** [comment]
I like this idea.  I personally would like to transfer ownership from my brothers account to mine. He no longer plays, and i'd like to get full access on my own account without losing player webshop progress. (tokens, time, achievments)

**Comment by mvaessen** [comment]
created dev item #956
